### PR TITLE
Nap 147 create terminal form

### PR DESCRIPTION
**Pull request by rckopteri**

### CODE REVIEW

File: database/src/main/resources/db/migration/V1_7__add_contact_details_for_services.sql [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ADD ATTRIBUTE homepage VARCHAR(1024);
+
+-- Add contact details to brokerage_provider_informaton
+ALTER TYPE "brokerage_provider_informaton"
+    ADD ATTRIBUTE "contact-address" address,
+    ADD ATTRIBUTE "contact-phone" VARCHAR(16),
+    ADD ATTRIBUTE "contact-gsm" VARCHAR(16),
+    ADD ATTRIBUTE "contact-email" VARCHAR(200),
+    ADD ATTRIBUTE homepage VARCHAR(1024);
+
```
Comment (tatut) [inline database/src/main/resources/db/migration/V1_7__add_contact_details_for_services.sql]: Move contact info to "transport-service" table, not in each service type?

File: ote/src/clj/ote/palvelut/transport.clj [hunk 2]
```diff
@@ -88,11 +102,40 @@
...
-            (upsert! db ::t-service/transport-service value)]
+         (let [new-value (dissoc value :transport-service)
+               transport-service (upsert! db ::t-service/transport-service new-value)]
         (places/link-places-to-transport-service!
          db (::t-service/id transport-service) places)))))
 
+(defn- save-terminal-info
+  "UPSERT! given data to database. "
+  [db data]
+  (println "Terminal DATA: " (pr-str data))
```
Comment (tatut) [inline ote/src/clj/ote/palvelut/transport.clj]: Add access controls here. User must be associated to the transport-operator they are trying to add a service for.

File: ote/src/clj/ote/services/places.sql [hunk 3]
```diff
@@ -17,3 +17,27 @@ VALUES (:transport-service-id,
            FROM places
           WHERE id = :place-id),
         true);
+
+-- name: save-point-for-transport-service!
+INSERT INTO operation_area
+       ("id", "transport-service-id", "location", "primary?")
+VALUES (
+    :id,
```
Comment (tatut) [inline ote/src/clj/ote/services/places.sql]: Indentation, fields to the right side of the "river".

File: ote/src/clj/ote/palvelut/transport.clj (hunk 2 above)
Comment (rckopteri) [reply ote/src/clj/ote/palvelut/transport.clj]: Created new ticket for this

**Pull request merged**